src/backend/joanie/debug/views.py

```diff
@@ -249,3 +259,56 @@ def get_basic_document_context(self, **kwargs):
                 "product": {"name": "deploy turn-key partnerships", "description": ""},
             },
         }
+
+
+@method_decorator(csrf_exempt, name="dispatch")
+class DebugPaymentTemplateView(TemplateView):
+    """
+    Simple class to render the payment form
+    """
+
+    # model = None
+    # issuer_document = None
+    template_name = "debug/payment.html"
+
+    def get_context_data(self, **kwargs):
+        """
+        Base method to prepare the document to render in the debug view in base64.
+        """
+        context = super().get_context_data()
+        backend = LyraBackend(
+            {
+                "username": "69876357",
+                "password": "testpassword_DEMOPRIVATEKEY23G4475zXZQ2UA5x7M",
```

Comment:
Should we get those values payment.configuration ?

Reply:
I don't know if we want to use real payment credentials in this view.
That's why I hardcoded provided test values, WDYT ?